packages/@uppy/xhr-upload/src/index.js

```diff
@@ -112,7 +114,7 @@ module.exports = class XHRUpload extends Plugin {
        * @param {string} responseText the response body string
        * @param {XMLHttpRequest | respObj} response the response object (XHR or similar)
```

Comment:
If we are removing the parameter, shouldn't we also remove them in the JSDoc comment?